tcf_website/management/commands/load_grades.py

```diff
@@ -15,16 +14,50 @@ class Command(BaseCommand):
     in the tcf_website/management/commands/grade_data/csv/ directory.
     This should take ~20 min to run to load all the grades
     """
-    help = 'Imports grade data from VAGrades CSV files into PostgresSQL database'
+    help = 'Imports FOIAed grade data files into PostgreSQL database'
 
-    # Not a good practice but declared as global for readability & convenience?
-    global course_grades
-    global course_instructor_grades
+    def __init__(self, stdout=None, stderr=None, no_color=False, force_color=False):
+        super().__init__(stdout, stderr, no_color, force_color)
 
-    course_grades = {}
-    course_instructor_grades = {}
+        # Initialize some variables that get used everywhere
+
+        # Dict mapping instructor first/last name to their ID in our database
+        # We used to be able to include email, but the UVA dataset doesn't include it!
+        # This is pretty risky if instructors have the exact same name, so yikes
+        # TODO: figure out how to differentiate (maybe by classes taught?)
+        self.instructors = {
+            (obj['first_name'], obj['last_name']): obj['id']
+            for obj in Instructor.objects.values('id', 'first_name',
```

Comment:
Looks like [Joe looked into this](https://github.com/thecourseforum/theCourseForum2/pull/434#discussion_r779214887) and it seems like the way our load semester script works makes it impossible for there to be duplicates (because instructors with the same name are all consolidated into the same Instructor object lol). The SIS API does support providing instructor emails, so when (/if lol) that gets reworked, then we'll have to revisit this.